subsys/bluetooth/controller/ll_sw/nordic/lll/lll_sync_iso.c

```diff
@@ -289,6 +292,35 @@ static int prepare_cb_common(struct lll_prepare_param *p)
 	crc_init[0] = lll->bis_curr;
 	(void)memcpy(&crc_init[1], lll->base_crc_init, sizeof(uint16_t));
 
+#if defined(CONFIG_BT_CTLR_SYNC_ISO_SLOT_WINDOW_JITTER)
+	if (p->ticks_drift) {
+		uint32_t drift_us;
+		uint8_t skipped;
+
+		drift_us = HAL_TICKER_TICKS_TO_US(p->ticks_drift);
+
+		skipped = DIV_ROUND_UP(drift_us, lll->sub_interval);
+		if (skipped >= lll->irc) {
+			skipped = lll->irc;
+		}
+
+		lll->irc_curr += skipped;
+
+		while (skipped--) {
+			/* Calculate the radio channel to use for subevent */
+			data_chan_use = lll_chan_iso_subevent(data_chan_id,
+						lll->data_chan_map,
+						lll->data_chan_count,
+						&lll->data_chan_prn_s,
+						&lll->data_chan_remap_idx);
+		}
+
+		drift_us %= lll->sub_interval;
+		drift_us = lll->sub_interval - drift_us;
```

Comment:
I assume that `sub_interval` is also in `us`? 

But what is this doing? This looks like pure math magic to me - Can you add a comment what, why or how you are calculating a `drift_us` based on this?

Reply:
i am trying to calculate a remaining `sub_interval` of a remainder (modulus) of `sub_interval` of `drift_us`... now you see I am not good at explaining :-(...